guide/additional-info/changes-in-v12.md

```diff
@@ -553,29 +553,29 @@ There have been several changes made to the `ClientOptions` object located in `c
 
 #### ClientOptions#shards
 
-`clientOptions.shards` has been removed and is functionally equivalent to `clientOptions.shardCount` on v12.
+`clientOptions.shards` was removed and is functionally equivalent to `clientOptions.shardCount` on v12.
 
 #### ClientOptions#sync
 
-`clientOptions.sync` has been removed entirely, along with all other user account-only properties and methods.
+`clientOptions.sync` was removed entirely, along with all other user account-only properties and methods.
 
 #### ClientOptions#disabledEvents
 
-`clientOptions.disabledEvents` has been removed in favor of using intents. Please refer to our more [detailed article about this topic](/popular-topics/intents)
+`clientOptions.disabledEvents` was removed in favor of using intents. Please refer to our more [detailed article about this topic](/popular-topics/intents)
 
 ### ClientUser
 
 #### ClientUser#acceptInvite
 
-`clientUser.acceptInvite()` has been removed entirely, along with all other user account-only properties and methods.
+`clientUser.acceptInvite()` was removed entirely, along with all other user account-only properties and methods.
 
 #### ClientUser#addFriend
 
-`clientUser.addFriend()` has been removed entirely, along with all other user account-only properties and methods.
+`clientUser.addFriend()` was removed entirely, along with all other user account-only properties and methods.
 
 #### ClientUser#avatarURL
 
-`clientUser.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. The `dynamic` option allows you to always get a `.gif` file for animated avatars. Otherwise the returned link will fall back to the format specified in the `format` option or `.webp` (it's default) if none is provided.
+`clientUser.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. The `dynamic` option will enable you always to get a `.gif` file for animated avatars. Otherwise, the returned link will fall back to the format specified in the `format` option or `.webp` (it's default).
```

Comment:
```suggestion
`clientUser.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If you provide the `dynamic` option, you will receive a `.gif` URL if the image is animated; otherwise, it will fall back to the specified `format` or its default `.webp`.
```
To match the text from `clientUser#displayAvatarURL`